packages/process/package.json

```
@@ -11,33 +11,26 @@
     "directory": "packages/process"
   },
   "author": "Frontside Engineering <engineering@frontside.com>",
-  "license": "MIT",
-  "files": [
-    "README.md",
-    "CHANGELOG.md",
-    "dist/**/*",
-    "src/**/*"
-  ],
+    "license": "MIT",
```

Is the indenting off here?
```suggestion
  "license": "MIT",
```